open-apparel-registry/open-apparel-registry
# Add downloadable XLSX contributor template

Contributors regularly run into CSV uploading issues. We have support for uploading XLSX files so this commit extends that support by replacing the programatically generated CSV template with static, downloadable XLSX and CSV template files.

## Overview

Brief description of what this PR does, and why it is needed.

Connects #714 

## Demo

<img width="688" alt="Screen Shot 2019-11-11 at 2 30 20 PM" src="https://user-images.githubusercontent.com/17363/68622641-de062300-048f-11ea-825e-c7452f588489.png">

## Testing Instructions

* Sign in as "c2@example.com:password"
* Browse http://localhost:6543/contribute. Verify that there are 2 download buttons.
* Click both download buttons and verify that they successfully link to XLSX and CSV templates respectively.

## Checklist
